pkg/controller/backup_sync_controller.go

```diff
@@ -134,6 +136,32 @@ func (c *backupSyncController) run() {
 
 	for _, location := range locations {
 		log := c.logger.WithField("backupLocation", location.Name)
+
+		if backupSyncPeriod, ok := location.Spec.Config[backupSyncPeriodKey]; ok {
```

Comment:
I think if we're adding this to BackupStorageLocations, it makes more sense to be a first-class `Spec` field (optional) than an opaque config key, since it's not provider-specific. It would also mean it could be strongly typed as a duration field. Thoughts?

Reply:
Agreed, I did find this strange. `velero install` should also set the default value when creating a BSL